### PR TITLE
Multiple CTAs + Hyperlinks for Fullscreen Marquee

### PR DESCRIPTION
**Pull request by echen-adobe**
Describe your specific features or fixes

Changes the fullscreen marquee so that it detects multiple CTAs and if the CTAs are italicized transforms them into hyperlinks.

Resolves: [MWPW-152925](https://jira.corp.adobe.com/browse/MWPW-152925)

Test URLs:
- Before: https://main--express--adobecom.hlx.page/express/
- After: https://fullscreen-marquee-cta--express--adobecom.hlx.page/drafts/echen/flyer-copy

Screenshot 2024-07-16 at 12 46 00 PM

### CODE REVIEW

File: express/blocks/fullscreen-marquee/fullscreen-marquee.js [hunk 1]
```diff
@@ -115,23 +136,19 @@ async function buildApp(block, content) {
...
-  if (cta) {
-    cta.classList.add('xlarge');
-
-    const highlightCta = cta.cloneNode(true);
-    const appHighlight = createTag('a', {
-      class: 'fullscreen-marquee-app-frame-highlight',
-      href: cta.href,
+function decorateMultipleCTAs(block) {
+  const links = Array.from(block.querySelectorAll('a')).slice(1)
+  links
```
Comment (fullcolorcoder) [inline express/blocks/fullscreen-marquee/fullscreen-marquee.js:145]: Could this be on one line?

File: express/blocks/fullscreen-marquee/fullscreen-marquee.js [hunk 2]
```diff
@@ -115,23 +136,19 @@ async function buildApp(block, content) {
...
+    .forEach((link) => {
+      if (link.classList.contains('button') && link.closest('em')) {
+        link.classList.remove('button')
```
Comment (fullcolorcoder) [inline express/blocks/fullscreen-marquee/fullscreen-marquee.js:148]: semicolon;

File: express/blocks/fullscreen-marquee/fullscreen-marquee.js [hunk 3]
```diff
@@ -115,23 +136,19 @@ async function buildApp(block, content) {
...
+        link.classList.add('hyperlink')
```
Comment (fullcolorcoder) [inline express/blocks/fullscreen-marquee/fullscreen-marquee.js:149]: No test coverage for these lines.

**Review by fullcolorcoder — CHANGES_REQUESTED** [review]
Small formatting things mostly. 

Is this to supplement the original ask in the ticket? I saw this screenshot in there. 
image

Also, is it expected that the CTAs disappear on mobile?

**Comment by echen-adobe** [comment]
> Small formatting things mostly.
> 
> Is this to supplement the original ask in the ticket? I saw this screenshot in there. image
> 
> Also, is it expected that the CTAs disappear on mobile?

Hey Brad it is expected for the buttons to disappear on mobile because the block is designed to have an entirely different design based on the screen resolution. 

The screenshot you provided is from a different ticket.

File: express/blocks/fullscreen-marquee/fullscreen-marquee.js (hunk 1 above)
Comment (echen-adobe) [reply express/blocks/fullscreen-marquee/fullscreen-marquee.js:150]: Line 144 is already its own thing. if you are talking about lines 144 - 156 then no, it cannot be done in a single line.

**Review by fullcolorcoder — APPROVED** [review]

**Pull request merged**